algosdk/kmd.py

```diff
@@ -35,6 +35,7 @@ def kmd_request(self, method, requrl, params=None, data=None):
             requrl (str): url for the request
             params (dict, optional): parameters for the request
             data (dict, optional): data in the body of the request
+            timeout (int, optional): request timeout in seconds
```

Comment:
Let's document the 30 second default here.

Reply:
Would be inconsistent since "response_format" is default "json" and is not documented in the docstring either

Reply:
Agreed, the signature shows in the docs, so the defaults are visible.